docs/dev/connectors/streamfile_sink.md

```diff
@@ -110,12 +110,45 @@ interactions of bucket assigners and rolling policies.
 
 In the above example we used an `Encoder` that can encode or serialize each
 record individually. The streaming file sink also supports bulk-encoded output
-formats such as [Apache Parquet](http://parquet.apache.org). To use these,
-instead of `StreamingFileSink.forRowFormat()` you would use
+formats such as [Apache Parquet](http://parquet.apache.org) or [Apache Avro](https://avro.apache.org/).
```

Comment:
The `flink-avro` package has an Avro row encoded format, meaning Avro itself is row encoded.

You can use Avro format definition to create parquet writers, but that is still the parquet format then, it only uses Avro as the schema definition.

Reply:
The Avro is row-encoded, but if you write it to a file, it is still in blocks: https://avro.apache.org/docs/1.8.2/spec.html#Object+Container+Files

Avro includes a simple object container file format. A file has a schema, and all objects stored in the file must be written according to that schema, using binary encoding. Objects are stored in blocks that may be compressed. 
A file data block consists of:

- A long indicating the count of objects in this block.
- A long indicating the size in bytes of the serialized objects in the current block, after any codec is applied
- The serialized objects. If a codec is specified, this is compressed by that codec.
- The file's 16-byte sync marker.